src/org/zaproxy/zap/extension/scripts/resources/help/contents/console.html

```diff
@@ -20,6 +20,9 @@ <H1>Script Console Tab</H1>
 <p>
 Right-click in the script area for display and editing options.
 <p>
+in ZAP versions after 2.7.0 if the script currently displayed in the console is changed by another program then you 

```

Comment:
`In` (first upper case).

Reply:
We should also mention the new behaviour (that ZAP will now pick/use the latest file changes).